### PR TITLE
Pin lock grace period set to 0

### LINKED ISSUE #1732: Set the App Lock grace period to 0
like on iOS: https://github.com/vector-im/element-x-ios/pull/2014

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : Change pin code grace period

## Content

Set PIN code grace period to 0

## Motivation and context

Closes #1732

## Tests

- Configure PIN code
- Put the app into BG
- Open the app again
- Observe that the user is prompted to enter their PIN code.

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

File: appconfig/src/main/kotlin/io/element/android/appconfig/LockScreenConfig.kt [hunk 1]
```diff
@@ -72,7 +73,7 @@ object LockScreenConfigModule {
         pinBlacklist = setOf("0000", "1234"),
         pinSize = 4,
         maxPinCodeAttemptsBeforeLogout = 3,
-        gracePeriod = 90.seconds,
```
Comment (jmartinesp) [inline appconfig/src/main/kotlin/io/element/android/appconfig/LockScreenConfig.kt:75]: Could we move this value to `LockScreenConfig` too?

Comment (bmarty) [reply appconfig/src/main/kotlin/io/element/android/appconfig/LockScreenConfig.kt:75]: We are using a Module here to be able to inject different instances of LockScreenConfig in test. As discussed with @ganfra it was the best solution to do put default values for this object.

Comment (jmartinesp) [reply appconfig/src/main/kotlin/io/element/android/appconfig/LockScreenConfig.kt:75]: Oh, my bad. I didn't realise this **was** in `LockScreenConfig` 🤦 .

**Review by jmartinesp — APPROVED** [review]
LGTM, sorry for the misunderstanding with `LockScreenConfig`.

**Pull request merged**